app/services/calculators/dependant_allowance_calculator.rb

```diff
@@ -0,0 +1,71 @@
+module Calculators
+  class DependantAllowanceCalculator
+    def self.call(dependant)
+      new(dependant).call
+    end
+
+    def initialize(dependant)
+      @dependant = dependant
+    end
+
+    def call
+      return child_under_15_allowance if under_15_years_old?
+
+      return child_aged_15_allowance - monthly_income if under_16_years_old?
```

Comment:
Could this be extracted to a`child_under_16_allowance` to make this easier to read?